qsr_lib/src/qsrlib_qsrs/qsr_arg_relations_distance.py

```diff
@@ -101,10 +102,20 @@ def make(self, *args, **kwargs):
         for t in input_data.get_sorted_timestamps():
             world_state = input_data.trace[t]
             timestamp = world_state.timestamp
-            if kwargs["qsrs_for"]:
-                qsrs_for, error_found = self.check_qsrs_for_data_exist(world_state.objects.keys(), kwargs["qsrs_for"])
-            else:
-                qsrs_for = self.qsrs_for_default(world_state.objects.keys())
+            qsrs_for_from_args, error_found = True, False
+            try:
+                qsrs_for = kwargs["dynamic_args"][self._unique_id]["qsrs_for"]
+            except KeyError:
+                try:
+                    qsrs_for = kwargs["qsrs_for"]
+                except KeyError:
+                    qsrs_for = self.qsrs_for_default(world_state.objects.keys())
+                    qsrs_for_from_args = False
```

Comment:
As I said in #114 this is too confusing. Do either `kwargs["dynamic_args"][self._unique_id]["qsrs_for"]` or `kwargs["qsrs_for"]` but not both.